### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by luisfff29**

### CODE REVIEW

File: list2.py [hunk 1]
```diff
@@ -35,12 +35,17 @@ def remove_adjacent(nums):
 # The solution should work in "linear" time, making a single pass of both lists.
 # Hint: Don't use `sort` or `sorted` -- they are not linear time.
 def linear_merge(list1, list2):
-    """Your code goes here.  Edit this docstring."""
-    return
-
+    list = list1 + list2
```
Comment (madarp) [inline list2.py]: Watch out here.  You are using python's built-in 'list' keyword as a variable, and you are changing the functionality by doing this ..

File: list2.py [hunk 2]
```diff
@@ -18,8 +18,16 @@
 
 
 def remove_adjacent(nums):
-    """Your code goes here.  Edit this docstring."""
-    return
+    list = []
```
Comment (madarp) [inline list2.py]: using `list` as a variable is not good for python -- you are changing default behavior ..  pick variable names that do not collide with Python's reserved keywords, such as `list_`

**Comment by madarp** [comment]
Terrific, this is the way to do it.  Small commits with short, meaningful commit messages.
HOWEVER, please make sure you are passing all the tests.

File: list2.py (hunk 1 above)
Comment (luisfff29) [reply list2.py]: OMG! that's why 'list' was always on green lol.
I'll change it right now. Thank you!

File: list2.py (hunk 2 above)
Comment (luisfff29) [reply list2.py]: I appreciate it 💯 
I'll use that.

**Pull request merged**